### PR TITLE
Card Grid & Card

### PR DESCRIPTION
**Pull request by isastettler**
[Jira Ticket](https://jira.nypl.org/browse/RENO-2901)

**This PR does the following:**
- adds LinkCard and LinkCardGrid Components
- adds chakra styling file for LinkCard and LinkCardGrid
- replaces CardGrid with LinkCardGrid and LinkCard
- adds Discover, From Our Blog, Updates section using LinkCardGrid and LinkCard

### Review Steps:
- [ ] [Tugboat](https://pr269-flv7fqmwqh8frbberjjzgfvqler2js0h.tugboat.qa/)

### Front End Review Steps:
- [ ] View [Example](http://localhost:3000/)

### CODE REVIEW

**Comment by nypl-wluisi** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr269-flv7fqmwqh8frbberjjzgfvqler2js0h.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62586972bda4981539f76633

File: src/components/home-v1/LinkCard.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  date?: string;
+  description?: string;
+  location?: string;
+  image: string;
+  url?: string;
+}
+
+interface LinkCardProps {
+  item: LinkCardItem;
+  variant?: Variant;
```
Comment (nypl-wluisi) [inline src/components/home-v1/LinkCard.tsx:25]: Remove the `Variant` enum and just do union, i.e., 
```ts
variant?: "event-spotlight" | "event-card" | "blog-card" | "updates-card"
```

File: src/components/home-v1/LinkCard.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useStyleConfig } from "@chakra-ui/system";
+// Components
+import { Heading, Text, Link, Grid, GridItem, Image } from "@chakra-ui/react";
+
+export enum Variant {
```
Comment (nypl-wluisi) [inline src/components/home-v1/LinkCard.tsx:6]: See comment on line 25, but we can remove this

File: src/components/home-v1/LinkCardGrid.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import { Grid, GridItem } from "@chakra-ui/react";
+import ComponentWrapper from "./ComponentWrapper";
+import LinkCard, { Variant, LinkCardItem } from "./LinkCard";
+
+interface LinkCardGridProps {
+  title: string;
+  link: string;
+  items: LinkCardItem[];
+  hoverStyle?: boolean;
+  variant?: string;
```
Comment (nypl-wluisi) [inline src/components/home-v1/LinkCardGrid.tsx:13]: change `string` to a union: `"row-grid" | "column-grid" | "updates-grid"`

File: src/components/home-v1/LinkCard.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      </GridItem>
+      <GridItem>
+        <Heading as="h3">
+          <Link href={item.url}>{item.title}</Link>
+        </Heading>
+        <Link
+          href={item.url}
+          display="block"
+          //@ts-ignore
+          tabIndex="-1"
```
Comment (nypl-wluisi) [inline src/components/home-v1/LinkCard.tsx:51]: change this to `tabIndex={-1}` and remove the `//@ts-ignore`

**Review by nypl-wluisi — CHANGES_REQUESTED** [review]
In general this looks great, there's a few inline comments I added. Also, small renaming request for the components:

- Rename `LinkCard` → `Card` (and related props, theme obj, etc)
- Rename `LinkCardGrid` → `CardGrid` (and related props, theme obj, etc)

Move both components to a dir called `CardGrid`

File: src/components/home-v1/LinkCard.tsx (hunk 1 above)
Comment (isastettler) [reply src/components/home-v1/LinkCard.tsx:25]: I am curious about the reason or if this is a preference?

File: src/components/home-v1/LinkCard.tsx (hunk 4 above)
Comment (isastettler) [reply src/components/home-v1/LinkCard.tsx:51]: I made the same change in FeaturedEvents.tsx

**Comment by nypl-wluisi** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr269-flv7fqmwqh8frbberjjzgfvqler2js0h.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/62586972bda4981539f76633

**Comment by nypl-wluisi** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr269-mkx2j2otv2vuokgazu4cw9bshmywy58a.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/625ece2bef17c6f5698b0fa2

**Comment by nypl-wluisi** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr269-mkx2j2otv2vuokgazu4cw9bshmywy58a.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/625ece2bef17c6f5698b0fa2

**Review by nypl-wluisi — APPROVED** [review]
This looks good! The accessibility related stuff we can follow up with Zach next week. I'm going to merge this now so you can create a new branch off the `RENO-2761/homepg-prototype` branch for the Hero work.

**Pull request merged**